.github/workflows/ci.yml

```diff
@@ -48,8 +54,8 @@ jobs:
         run: composer install --no-progress --prefer-dist --optimize-autoloader
 
       - name: Code Analysis (PHP CS-Fixer)
-        if: matrix.code-analysis == 'yes'
-        run: php vendor/bin/php-cs-fixer fix --dry-run --diff
+        if: matrix.code-style == 'yes'
+        run: PHP_CS_FIXER_IGNORE_ENV=true php vendor/bin/php-cs-fixer fix --dry-run --diff
```

Comment:
This is handy to specify if we want to run the "old" php-cs-fixer with a more modern version of PHP.
It is not essential to specify now, because we are only running it with PHP 7.1 in CI. But I will leave it here so I have it in future, it does no harm.